### PR TITLE
gui/lang: correct lang typo and translation.

### PR DESCRIPTION
**Pull request by Croden1999**
correct lang typo and translation, and also add the "Full Screen" tip in control dialog (someone need this). if someone need me change some things, i'll check it.

plz Mr. @Zangetsu38 check this code.

### CODE REVIEW

File: lang/de.xml [hunk 1]
```diff
@@ -25,10 +25,10 @@
 	</main_menubar>
 
 	<about name="Über">
-		<vita3k>Vita3K: a PS Vita/PSTV Emulator. The world's first functional PS Vita/PSTV emulator.</vita3k>
+		<vita3k>Vita3K: a PS Vita/PS TV Emulator. The world's first functional PS Vita/PS TV emulator.</vita3k>
```
Comment (Zangetsu38) [inline lang/de.xml:28]: no sur for this ? 🤔  i have all time thinks is PSTV like PSVita

File: vita3k/gui/src/controls_dialog.cpp [hunk 2]
```diff
@@ -101,6 +101,7 @@ void draw_controls_dialog(GuiState &gui, EmuEnvState &emuenv) {
     ImGui::Separator();
     ImGui::Spacing();
     ImGui::TextColored(GUI_COLOR_TEXT_MENUBAR, "%-16s", lang["gui"].c_str());
+    ImGui::Text("%-16s    %-16s", lang["full_screen"].c_str(), "F11");
```
Comment (Zangetsu38) [inline vita3k/gui/src/controls_dialog.cpp:104]: ah yes good idea lol

File: lang/de.xml (hunk 1 above)
Comment (Croden1999) [reply lang/de.xml:28]: but in Asia it called "PS Vita TV", USA and EU are called "PS TV", so i think it is name different but they same is one thing.

Comment (Zangetsu38) [reply lang/de.xml:28]: oki i see

**Review by Zangetsu38 — APPROVED** [review]

**Pull request merged**